apps/prairielearn/src/question-servers/freeform.js

```diff
@@ -448,7 +450,9 @@ function checkData(data, origData, phase) {
       return 'invalid type: ' + String(type);
     }
     if (!editPhases.includes(phase)) {
-      if (!_.has(origData, prop)) return '"' + prop + '" is missing from "origData"';
+      if (!Object.prototype.hasOwnProperty.call(origData, prop)) {
+        return '"' + prop + '" is missing from "origData"';
```

Comment:
Nit: if we are cleaning this up, maybe change to a template literal?

Reply:
I can do that, sure.

Reply:
Done in 0ee9722.